TribalWars/src/ApplicationCore/Enums/MaterialType.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ApplicationCore.Enums
+{
+    public enum MaterialType
+    {
+        People,
```

Comment:
I think calling 'People' a 'Material' is kinda inappropriate.
Maybe 'Resource' would fit the intention better?